tests/run/annotate_html.pyx

```diff
@@ -11,6 +11,7 @@
 
 >>> import re
 >>> assert re.search('<pre .*def.* .*mixed_test.*</pre>', html)
+>>> assert not re.search('Complete\scythonized\scode', html) # per default no complete c code
```

Comment:
The first line in every annotated html is "Generated by Cython XXX" - quite close to "/* Generated by Cython ";)

 I have added an explicit marker rather than using a magic value - not sure it is much better though...